flink-kubernetes/src/main/java/org/apache/flink/kubernetes/utils/Constants.java

```diff
@@ -80,4 +84,14 @@
 	public static final String RESTART_POLICY_OF_NEVER = "Never";
 
 	public static final String NATIVE_KUBERNETES_COMMAND = "native-k8s";
+
+	// Constants for Kubernetes high availability
+	public static final String LEADER_ADDRESS_KEY = "address";
+	public static final String LEADER_SESSION_ID_KEY = "sessionId";
+	public static final String CHECKPOINT_COUNTER_KEY = "counter";
+	public static final String RUNNING_JOBS_REGISTRY_KEY_PREFIX = "runningJobsRegistry";
+	public static final String JOB_GRAPH_STORE_KEY_PREFIX = "jobGraph";
+
+	public static final String LOCK_IDENTITY = UUID.randomUUID().toString();
```

Comment:
Moving the `LOCK_IDENTITY` to the `KubernetesLeaderElectionService` makes sense to me.
However, I am not sure why do we need to make it non static. Actually, I want the lock identity to be same for different leader election service(e.g. Dispatcher, ResourceManager, etc.).

Reply:
I'm not sure about sharing the identity across leader election services. By making `LOCK_IDENTITY` static, we implicitly introduces a constraint that we cannot have multiple contenders in the same process that competing for leadership of the same service. This may not be a problem for Flink's current implementation, but I would suggest to not making such assumption if it doesn't add too much effort.

Reply:
I will make `LOCK_IDENTITY` non static and could be specified via `KubernetesLeaderElectionConfiguration`.